src/azul/health.py

```diff
@@ -0,0 +1,92 @@
+import boto3
+
+from botocore.exceptions import ClientError
+
+from azul import config
+
+
+def get_health_elasticsearch():
+    try:
+        es = boto3.client('es')
+        elasticsearch = es.describe_elasticsearch_domain(DomainName=config.es_domain)
+
+        return {
+            'elasticsearch': {
+                'status': 'UP',
+                'details': {
+                    'domain': elasticsearch['DomainStatus']['DomainName'],
+                    'endpoint': elasticsearch['DomainStatus']['Endpoint'],
+                },
+            }
+        }
+    except ClientError as ex:
+        return {
+            'elasticsearch': {
+                'status': 'DOWN',
+                'details': {
+                    'message': ex.response['Error']['Message'],
+                }
+            }
+        }
+
+
+def get_health_queues():
+    notify_queue = __get_health_notify_queue()
+    fail_queue = __get_health_fail_queue()
+
+    return {
+        'queues': [
+            notify_queue,
+            fail_queue,
+        ]
+    }
+
+
+def __get_health_notify_queue():
+    try:
+        sqs = boto3.resource('sqs')
+        notify_queue = sqs.get_queue_by_name(QueueName="azul-notify-{config.deployment_stage}".format(config=config))
+
+        return {
+            'notifyQueue': {
+                'status': 'UP',
+                'details': {
+                    'url': notify_queue.url,
+                    'approximateNumberOfMessages': notify_queue.attributes['ApproximateNumberOfMessages'],
+                }
+            },
+        }
+    except ClientError as ex:
+        return {
+            'notifyQueue': {
+                'status': 'DOWN',
+                'details': {
+                    'message': ex.response['Error']['Message'],
+                }
+            },
+        }
+
+
+def __get_health_fail_queue():
```

Comment:
This function is 99% identical to the one above it.

https://en.wikipedia.org/wiki/Don%27t_repeat_yourself